libs/secure/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
I think new common crypto library should be done as an implementation of a new shared model interfaces. Also it looks like it deserves a separate PR.